homeworks/03-data-structure/src/BinarySearchTree.test.ts

```diff
@@ -0,0 +1,106 @@
+import { BinarySearchTree } from "./BinarySearchTree"
+import { TreeNode } from "./BinaryTree"
+
+const binaryTreeFake = {
+  value: 4,
+  left: {
+    value: 2,
+    left: {
+      value: 1,
+    },
+    right: {
+      value: 3,
+    },
+  },
+  right: {
+    value: 5,
+  },
+}
+
+const fakeCustomCompareFunction = (a: number, b: number) => {
+  if (a === b) return 0
+  if (a > b) return -1
+  return 1
+}
+
+describe("BinaryTree", () => {
+  describe("has()", () => {
+    it("returns false for emprty tree", () => {
+      const binaryTree = new BinarySearchTree()
+      const expectedResult = false
```

Comment:
Looks redundant. You can write it here `toEqual(false)`, or better `toBe(false)` or much better `toBeFalse(false)`.